docs/xostor/xostor.md

```diff
@@ -520,6 +592,95 @@ Consider the following:
+    currently-attached ( RO): true
+
+
+uuid ( RO)                  : 1d872d5b-fb60-dbd7-58fc-555a211f18fa
+             host-uuid ( RO): ef942670-e37d-49e6-81d0-d2a484b0cd10
+               sr-uuid ( RO): d5f990f6-abca-0ebf-8582-b7e55901fb50
+         device-config (MRO): group-name: linstor_group/thin_device; redundancy: 2; provisioning: thin
+    currently-attached ( RO): true
+```
+
+Then if you want to fix an incorrect group name value or even add a new host, use this command with the correct `<GROUP_NAME>` and `<HOST_UUID>`:
+```
+xe host-call-plugin host-uuid=<HOST_UUID> plugin=linstor-manager fn=addHost args:groupName=<GROUP_NAME>
+```
+For a short description, this command (re)creates a PBD, opens DRBD/LINSTOR ports, starts specific services and adds the node to the LINSTOR database.
+
+A storage layer is not directly added to this node. You can verify the storage state like this:
+```
+linstor sp list
+```
+
+Small example:
+
+A `LVM_THIN` entry is missing for `hpmc17` in this context:
+```
+╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+┊ StoragePool                      ┊ Node   ┊ Driver   ┊ PoolName                  ┊ FreeCapacity ┊ TotalCapacity ┊ CanSnapshots ┊ State ┊ SharedName                              ┊
+╞══════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════╡
+┊ DfltDisklessStorPool             ┊ hpmc15 ┊ DISKLESS ┊                           ┊              ┊               ┊ False        ┊ Ok    ┊ hpmc15;DfltDisklessStorPool             ┊
+┊ DfltDisklessStorPool             ┊ hpmc16 ┊ DISKLESS ┊                           ┊              ┊               ┊ False        ┊ Ok    ┊ hpmc16;DfltDisklessStorPool             ┊
+┊ DfltDisklessStorPool             ┊ hpmc17 ┊ DISKLESS ┊                           ┊              ┊               ┊ False        ┊ Ok    ┊ hpmc17;DfltDisklessStorPool             ┊
+┊ xcp-sr-linstor_group_thin_device ┊ hpmc15 ┊ LVM_THIN ┊ linstor_group/thin_device ┊   476.66 GiB ┊    476.70 GiB ┊ True         ┊ Ok    ┊ hpmc15;xcp-sr-linstor_group_thin_device ┊
+┊ xcp-sr-linstor_group_thin_device ┊ hpmc16 ┊ LVM_THIN ┊ linstor_group/thin_device ┊   476.66 GiB ┊    476.70 GiB ┊ True         ┊ Ok    ┊ hpmc16;xcp-sr-linstor_group_thin_device ┊
+╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+```
+
+If VG/LV is missing on `hpmc17`, connect to this machine and use `vgcreate` with the wanted disks to create `linstor_group`. For `thick` configuration it's enough, for `thin` additional commands are required. Replace `linstor_group/thin_device` by your pool name if it's different in your configuration.
```

Comment:
```suggestion
If your host has disks that need to be added to the linstor SR, you will need to create VG/LV.
Use `vgcreate` with the wanted disks to create `linstor_group` on the host.
For `thin` additional commands are required, replace `linstor_group/thin_device` by your pool name if it's different in your configuration.
```

Reply:
I used part of your suggestion, feel free to reread and tell me what you think.